code/game/machinery/camera/tracking.dm

```diff
@@ -25,9 +25,10 @@
 	var/list/T = list()
 
 	for(var/obj/machinery/camera/C in L)
-		var/list/tempnetwork = C.network & src.network
-		if(tempnetwork.len)
-			T[text("[][]", C.c_tag, (C.can_use() ? null : " (Deactivated)"))] = C
+		if(C.network)
+			var/list/tempnetwork = C.network & network
+			if(tempnetwork && tempnetwork.len)
+				T[text("[][]", C.c_tag, (C.can_use() ? null : " (Deactivated)"))] = C

```

Comment:
> I wonder why some camera machines don't get their network set if the AI latejoins...

When the AI joins is unrelated; the problem is a camera (specifically, the one inside video cameras) getting its network list nulled out instead of set to an empty list. #6077 fixes it.